### ISSUE TITLE
Nothing works; is this a scam?

### ISSUE THREAD

**Issue by grantpowell** [Foliotek/Croppie#744]
I've literally spent HOURS copying, pasting, and editing different version of the "demos" to try and get them to work.

NOTHING WORKS!

I've spent much time editing and hacking the code in a desparate attempt to get the most BASIC implementation working.

A DEMO IMPLEMENTATION SHOULD LITERALLY TAKE 2 ACTIONS: 1) Copy, 2) Paste - WHY DOESN'T THIS WORK?!

What's the point of writing code, of building apps, that just. don't work.

A complete waste of time,.

This is the same experience I have VIRTUALLY EVERY SINGLE OPEN SOUCE CODE I EVER TRY TO USE>

In the end, I am almost always better off just building my own code from scratch; it takes less time and works better.

Ugh!

**Comment by grantpowell** [comment]
This is beyond frustrating! I am so disappointed. What a shame.

**Comment by grantpowell** [comment]
For those of you who want a working example, this person solved it - THANK YOU TO THIS PERSON!

Can you PLLEEEASEEEE update the documentation with something like this that actually works?!

https://stackoverflow.com/questions/41622389/using-croppie-simple-example

**Comment by bamdad-b** [comment]
If you are familiar with Html and js (which I assume you are) it doesn't take HOURS to arrange it. There are good examples in the blog page and it's simple and clear. I agree that this project needs to have a better documentation but it's still under development and has a lot of issues. I think you need to chill and be grateful.

**Comment by bret-miller** [comment]
It took some effort to find a JavaScript image cropper that actually did what I needed. I like Croppie. It works. I wish it were actively devloped, but it's not. Hopefully if it dies, something better will be available by then. If you are struggling with it, you might check out one of the other options:

https://www.dunebook.com/best-javascript-image-cropping-libraries/

**Comment by mhfs** [comment]
No, this is not a scam. Started using it for the first time today and it worked like a charm in 15 minutes.

Very grateful for all the effort by the contributors and community at large. Thank you.

**Comment by DominicGaribaldi** [comment]
We also had this working in minutes. Could it be better? Sure. FYI, that's what pull requests are for.

**Comment by tolmekian1453** [comment]
Yes, it works. Huge thanks to the dev.

> This is the same experience I have VIRTUALLY EVERY SINGLE OPEN SOUCE CODE I EVER TRY TO USE

You're doing something wrong then. JS can be weird to set up. It's confusing with all the `npm` and `bower` stuff going on. Just, don't blame the devs.

**Comment by arnav-kr** [comment]
got it in just 5 minutes it was that easy to setup

**Comment by Hanho-Kim** [comment]
Maybe because of the css. Importing Javascript is not enough to make it work.
